DynamicTablesPkg/Include/ArchCommonNameSpaceObjects.h

```diff
@@ -850,6 +852,55 @@ typedef struct CmArchCommonDbg2DeviceInfo {
   CHAR8              ObjectName[AML_NAME_SEG_SIZE + 1];
 } CM_ARCH_COMMON_DBG2_DEVICE_INFO;
 
+typedef struct CmArchCommonCxlHostBridgeInfo {

```

Comment:
Need a description header.

Also, would it be possible to add matching parsers in `DynamicTablesPkg/Library/Common/TableHelperLib/ConfigurationManagerObjectParser.c` ?